miden-crypto/src/merkle/mod.rs

```diff
@@ -27,6 +27,12 @@ pub use smt::{
     SMT_MAX_DEPTH, SMT_MIN_DEPTH, SimpleSmt, SimpleSmtProof, Smt, SmtLeaf, SmtLeafError, SmtProof,
     SmtProofError,
 };
+#[cfg(feature = "concurrent")]
+pub use smt::{
+    LargeSmt, LargeSmtError, MemoryStorage, SmtStorage, StorageUpdateParts, StorageUpdates, Subtree,
+};
+#[cfg(feature = "rocksdb")]
+pub use smt::{RocksDbConfig, RocksDbStorage};
```

Comment:
Not for this PR, but we should consider refactoring export organization of this module. It starts to feel very "flat". It may make sense to move a few things into submodules (e.g., maybe we should have `mmr` and `smt` submodules). Let's create an issue for this.